tests/test_parser.py

```diff
@@ -781,6 +781,75 @@ def function() -> int:
         assert parser.raw_statements == {1, 3, 4, 5, 6, 8, 9}
         assert parser.statements == {1, 8, 9}
 
+    def test_multiline_exclusion(self) -> None:
```

Comment:
This test could be broken into a number of separate tests since it's so long.

Reply:
Done, let me know if it should be done in a better way.